packages/scheduled-tasks/src/lib/strategies/ScheduledTaskRedisStrategy.ts

```diff
@@ -0,0 +1,71 @@
+import { container } from '@sapphire/framework';
+import type { Job, JobOptions, Queue, QueueOptions } from 'bull';
+import Bull from 'bull';
```

Comment:
```suggestion
import Bull, { Job, JobOptions, Queue, QueueOptions } from 'bull';
```

ESLint should've already errored on you that you were duplicating imports

Reply:
Hmm weird. It didnt